app/contributing/word-choice.md

```diff
@@ -28,7 +28,7 @@ Kong Mesh <br><br> Mesh | Kong's service mesh. Use "Kong Mesh" for the first men
 Kuma | Kong's open-source service mesh. <br><br> ❌&nbsp; Do not use "Kong Kuma". This is an open-source project supported by the CNCF and maintained, not owned, by Kong.
 Insomnia | Kong's open-source API client.
 Dev Portal <br><br> Dev Portal, self-managed <br> Dev Portal, cloud | A module for sharing APIs and their specs with developers, and enabling the developers to create applications based on Gateway or Konnect services. <br><br> ❌&nbsp; Do not use "Developer Portal".
-Service Hub | The service catalog in Konnect. <br><br> ❌&nbsp; Do not use "ServiceHub", "Service hub", or "Servicehub".
```

Comment:
Seeing descriptions like these make me feel more certain that we made the right decision with this rename!

The Konnect Dev Portal is a dev portal. The Konnect Service Catalog is a service catalog. 😄

Reply:
hard agree!